src/schema/service.ts

```diff
@@ -48,30 +52,26 @@ async function getAllSchemas(repository: SchemaRepository) {
 
 async function loadSchemas(
```

Comment:
Should we not also attempt to resolve the fully qualified schema?

Reply:
Eventually yes, but now it would require a bit more work and I don't think it's so benefical.

Reply:
Also slowing the re-deployment... so maybe when we have running service